rosco-core/src/test/groovy/com/netflix/spinnaker/rosco/providers/aws/AWSBakeHandlerSpec.groovy

```diff
@@ -628,7 +672,7 @@ class AWSBakeHandlerSpec extends Specification {
         aws_target_ami: targetImageName,
         repository: DEBIAN_REPOSITORY,
         package_type: BakeRequest.PackageType.DEB.packageType,
-        packages: fullyQualifiedPackageName,
+        packages: unqualifiedPackageName,
```

Comment:
What is the reason that this particular `package_name` parameter now results in an unqualified package name being returned?
It looks like it's just because of this change: `1 * imageNameFactoryMock.buildPackagesParameter(osPackages) >> unqualifiedPackageName`
But why is that change made to the particular test case?